tests/stdlib/trandom.nim

```diff
@@ -164,3 +164,15 @@ block: # random sample
       let stdDev = sqrt(n * p * (1.0 - p))
       # NOTE: like unnormalized int CDF test, P(wholeTestFails) =~ 0.01.
       doAssert abs(float(histo[values[i]]) - expected) <= 3.0 * stdDev
+
+
```

Comment:
regardless of the changes in lib/pure/times.nim (which may be desirable for their own sake), can't we fix the issue deterministically by using a thread local random state variable that gets reused? that way, we initialize it once (using sysrand or time), and then just update the state ?

Reply:
I see